gradle/libs.versions.toml

```diff
@@ -177,7 +178,8 @@ opentelemetry-api = { module = "io.opentelemetry:opentelemetry-api" }
 opentelemetry-bom = { module = "io.opentelemetry:opentelemetry-bom", version.ref = "opentelemetry" }
 opentelemetry-exporter-otlp = { module = "io.opentelemetry:opentelemetry-exporter-otlp" }
 opentelemetry-sdk = { module = "io.opentelemetry:opentelemetry-sdk" }
-opentelemetry-semconv = { module = "io.opentelemetry:opentelemetry-semconv", version.ref = "opentelemetryAlpha" }
+opentelemetry-alpha-bom = { module = "io.opentelemetry:opentelemetry-bom-alpha", version.ref = "opentelemetryAlpha" }
```

Comment:
Should we then use just the "alpha" bom for controlling OTel artefact versions?

Reply:
it would be technically possible yes but also kind of odd i.e. "why do we only depend on the alpha bom"

did you see the other discussion? need your input on semconv and why it is fine to depend on alpha versions for opentelemetry in the first place